src/app/page.tsx

```diff
@@ -1,49 +1,31 @@
-import { Card, CardContent, CardDescription, CardHeader, CardTitle } from "@/components/ui/card";
+import BankAccountsList from "@/components/bank-account/ba-list";
 import { getBankAccountsByUserId } from "@/domain/bankAccount-domain/ba-repository";
 import userService from "@/domain/user-domain/user-service";
-import { BankAccount } from "@prisma/client";
 import { headers } from "next/headers";
-import Image from "next/image";
-import Link from "next/link";
 import { redirect } from "next/navigation";
 
 export default async function Home() {
-  const session = await userService.server.getSession(await headers());
+  const apikeyHeaders = await headers();
```

Comment:
I don't think that name `apiKeyHeaders` - it is storing all headers and it is not connected to apiKey at all (I think).